source/docs/contracts_configuration.md

```diff
@@ -295,6 +340,22 @@ contracts: {
 ...
 ```
 
+### Specifying a tracking file
+
+In addition to enabling and disabling tracking, it's also possible to tell Embark which file it should use for tracking. This can be useful for tracking deployed Smart Contracts on different platforms, such as testnets and the mainnet. The tracking state of those platforms should most likely be under version control. Specifying a file gives us full control over that:
```

Comment:
I would specify that the file needs to have the structure above and/or should be like chains.json

Reply:
Can you elaborate on what you mean by "specifying"? Reading the code it seems the users doesn't have to worry about that as Embark will take care of everything.

Reply:
Oh I just mean in the paragraph above, to specify that the file (`it's also possible to tell Embark which file it should use for tracking`) needs to be chains.json file.

Reply:
I still don't follow.

The user actually doesn't even have to know what the contents of that file look like. If it doesn't exist, Embark is going to create it. So if I want the tracking state for Ropsten be put in a file `./ropsten.json` I just do

```
track './ropsten.json'
```

Once I deploy to Ropsten, Embark will put the tracking information in that file (assuming it doesn't exist yet). 

I never have to know what the contents look like.

Reply:
I don't think people will write these files by hand and put them into version control, if that's what you're referring to..

Reply:
oooh, ok my bad. I, myself, misunderstood the feature. I thought this worked that you give Embark a chains.json file and it uses that isn't of writing it. 

I forgot that it's a ctually just so it uses a different file/name for different environments.

Reply:
Ah, now it makes sense! :D